content/swiftui/concepts/views/terms/toggle/toggle.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Toggle'
+Description: 'A switch that can be turned on or off.'
+Subjects:
+  - 'Software Development'
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Toggle'
+  - 'Views'
+CatalogContent:
+  - 'learn-swiftui'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **`Toggle`** is a View that chooses between "on or off" options based on a `Bool` type value.
+
+## Syntax
+
+```pseudo
+Toggle(_ title: StringProtocol, isOn: Binding<Bool>)
+```
+
+The usual method for creating a text-only `Toggle` involves passing a `titile` string as the first parameter, rather than using a trailing [closure](https://www.codecademy.com/resources/docs/swift/closures), and a `Bool` type property `isOn:` that determines the state of the toggle (on or off).
```

Comment:
```suggestion
The usual method for creating a text-only `Toggle` involves passing a `title` string as the first parameter, rather than using a trailing [closure](https://www.codecademy.com/resources/docs/swift/closures), and a `Bool` type property `isOn:` that determines the state of the toggle (on or off).
```